config/routes.yaml

```
@@ -18,6 +18,11 @@ app_register:
     controller: App\Controller\RegistrationController::register
     methods: GET|POST
 
+app_account_activation:
+    path: /account-activation/{id}/{token}
```

```suggestion
    path: /account-activation/{id}/{token}
```

It's not secure to provide the id. Use email instead. And pass arguments by get params.

---

Is it better like that ? I used username instead of id not to expose it for security reason. But i'm not sure to fully understand what do you mean about "to pass arguments by get params" : in my controller i pass $username and $token as parameters of the method called to retrieve it from the request. The result is the same as if i do $request->get('param_key'); instead in this method. Is it what you mean ?